src/vrl/selectielijst/tests/test_command.py

```diff
@@ -0,0 +1,236 @@
+             'volksgezonheid_en_milieu': False,
+             'vhrosv': False,
+             'heffen_belastingen': False,
+             'alle_taakgebieden': False,
+             'procestermijn_opmerking': '',
+             }
+        )
+
+    def test_prepare_resultaat_specifiek(self):
+        """ test prepare_procestype function: cols for specifiek resultaat"""
+        proces_type = ProcesType.objects.create(**prepare_procestype(self.raw))
+        generiek_resultaat = Resultaat.objects.create(**prepare_resultaat(self.raw))
+        specifiek = prepare_resultaat(self.specifiek)
+        self.assertEqual(
+            specifiek['generiek_resultaat'],
+            generiek_resultaat
+        )
+        self.assertEqual(
+            specifiek['nummer'],
+            2
+        )
+
+    def test_read_xls_row_to_dict_read(self):
+        """ test read_xls_row_to_dict generator: on reading and converting to dict first row of data in xls file"""
+        sheet_iter = read_xls_row_to_dict(self.sheet)
+        first_row = next(sheet_iter)
+        self.assertDictEqual(
+            first_row,
+            self.raw
+        )
+
+    def test_command_no_argument(self):
+        """  test command: fails if no arguments are provided """
+        self.assertRaises(Exception, call_command)
+
+    def test_command_save_procestype(self):
+        """ test handle method: test if could be created model object after prepare_procestype function"""
+        proces_type = ProcesType.objects.create(**prepare_procestype(self.raw))
+        self.assert_(True)
+
+    def test_command_save_resultaat_generiek(self):
+        """ test handle method: if could be created model object after prepare_resultaat function"""
+        proces_type = ProcesType.objects.create(**prepare_procestype(self.raw))
+        generiek_resultaat = Resultaat.objects.create(**prepare_resultaat(self.raw))
+        self.assert_(True)
+
+    def test_command_resultaat_specifiek(self):
+        """ test handle method: test if could be created model object after prepare_resultaat function for specifiek case"""
+        proces_type = ProcesType.objects.create(**prepare_procestype(self.raw))
+        generiek_resultaat = Resultaat.objects.create(**prepare_resultaat(self.raw))
+        specifiek_resultaat = Resultaat.objects.create(**prepare_resultaat(self.specifiek))
+        self.assert_(True)
+
+    def test_command_success(self):
+        """ test handle method: read test data and write them into model. Check number of obs"""
+        call_command('load_data_from_excel', TESTDATA_FILENAME)
+        # for Resultaat
+        self.assertEqual(
+            len(Resultaat.objects.all()),
+            self.sheet.nrows - 1  # exclude the header
+        )
+        # for ProcesType - count distinct nummer
+        nummers = [self.sheet.cell(i, 0).value for i in range(1, self.sheet.nrows)]
+        self.assertEqual(
+            len(ProcesType.objects.all()),
+            len(set(nummers))
```

Comment:
see remark above - you're in control of the testdata, so you can make a very explicit, simple, assertion. Having to process this while debugging a future-breaking test leads to cognitive overload

Reply:
refactored the testcase